src/main/java/org/example/Hangman/Hangman.java

```diff
@@ -0,0 +1,99 @@
+package org.example.Hangman;
+
+import java.util.HashSet;
+import java.util.Scanner;
+import java.util.Set;
+
+public class Hangman {
+
+        public static final int WINNING_SCORE =3;
+
+    public void start() {
+
+
+        Player player1 = new Player();
+        Player player2 = new Player();
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("select first players name");
+        player1.setName(scanner.next());
+        player1.setScore(0);
+
+        System.out.println("select second players name");
+        player2.setName(scanner.next());
+        player2.setScore(0);
+
+        boolean endGame = false;
+
+        while(!endGame){
+            printScore(player1,player2);
+            runRuond(player1,scanner);
+            printScore(player1,player2);
+            runRuond(player2,scanner);
+
+            if(player1.getScore()==WINNING_SCORE || player2.getScore()==WINNING_SCORE){
+                endGame = true;
+                if(player1.getScore()==player2.getScore()){
+                    System.out.println("draw");
+                } else if (player1.getScore()==WINNING_SCORE) {
+                    System.out.println("P1 wins");
+                } else {
+                    System.out.println("P2 wins");
+                }
+
+            }
+        }
+        scanner.close();
+    }
+
+    private void printScore(Player player1, Player player2) {
+        System.out.printf("P1 score: %d\n",player1.getScore());
+        System.out.printf("P2 score: %d\n",player2.getScore());
+    }
+
+
+    public void runRuond(Player player,Scanner scanner){
+        player.setChances(7);
+        Set<Character> letters = new HashSet<>();
+
+        WordProccessor wordProccessor = new WordProccessor();
+        boolean currentRound = true;
+        wordProccessor.setNewWord();
+        wordProccessor.setNewArray();
+        String hashedWord = wordProccessor.hash(wordProccessor.getWordToArray());
```

Comment:
moze czegos nie rozumiem ale wydawalo mi sie ze w WordProcessor trzymasz slowo startowe i potem w razie postepow "zahaszowane" wiec ten stan jest tam. po co wiec ten string hasedhWord tutaj :D?

Reply:
No właśnie chodzi o to że zrobiłem pewne uproszczenie na początku i brnąłem w to. Wydaje mi się że wszystkie komentarze powyżej są związane z tym i jak coś zacznę poprawiać, to zgubię trochę moich założeń. Pytanie czy to bardzo źle gdyby teraz to tak zostało? Nie mam na to pomysłu teraz, a jeszcze chciałem dodać dwie rzeczy do zajęć.

Reply:
moze tak zostac :D powaznych bledow nie bylo zadnych